docker-compose.yml

```diff
@@ -180,7 +180,7 @@ services:
       GF_SERVER_ROOT_URL: "https://%(domain)s/grafana/"
       GF_SMTP_ENABLED: "${TTN_DASHBOARD_GRAFANA_SMTP_ENABLED:-false}"
       GF_SMTP_SKIP_VERIFY: "${TTN_DASHBOARD_GRAFANA_SMTP_SKIP_VERIFY:-false}"
-      GF_SMTP_HOST: "${TTN_DASHBOARD_MAIL_HOST_NAME:-postfix}:25"
+      GF_SMTP_HOST: "postfix:25"
```

Comment:
This is OK, but now it seems that TTN_DASHBOARD_MAIL_HOST_NAME is used only one place, and there is no documentation on how it should be used. I'll file another bug on that.